ExceptionsCases.cs

```diff
@@ -0,0 +1,202 @@
+﻿using HelloWorld.Models;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace HelloWorld
+{
+    public class ExceptionsCases
+    {
+        private static readonly log4net.ILog log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
```

Comment:
move logging functionality into the separate file and wrap logging methods into own ones